styles.css

```diff
@@ -269,6 +269,103 @@ li {
   width: 100%;
 }
 
+.popup-window {
+  width: 0;
+  height: 0;
+  position: fixed;
+  display: flex;
+  justify-content: center;
+  top: 0;
+  left: 0;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  z-index: -3;
+  opacity: 0;
+  padding: 0;
+  overflow-y: scroll;
+}
+
+.popup-show {
+  z-index: 100;
+  opacity: 1;
+  background: #c1c7d0;
+  width: 100%;
+  height: auto;
+  position: absolute;
+}
```

Comment:
- [ ] Hi @Alejandro-Bernal-M, great effort working on the pop-up modal section of your project, 👍 👍  but currently the blur effect given to the main page when the pop-up card is displayed is not implemented and this makes your work looks different from what we have in the Figma design. you can simply make use of the RGBA color format for the background-color or target the container holding the pop-up cards and give it a `backdrop-filter: blur(10px)` CSS property.

Your Design 👇 

![alex](https://user-images.githubusercontent.com/78634841/216170257-b6b4934e-1f41-47e7-babf-bdce72c59651.png)


>N.B: Please kindly implement this changes for both `mobile` and `desktop` version

Reply:
Thank you for the correction. I appreciate your recommendation and your effort.